pandas/tests/frame/test_reshape.py

```diff
@@ -765,7 +765,7 @@ def test_unstack_unused_level(self, cols):
         tm.assert_frame_equal(result, expected)
 
     def test_unstack_nan_index(self):  # GH7466
-        cast = lambda val: "{0:1}".format("" if val != val else val)
+        cast = lambda val: f"{'' if val != val else val:1}"
```

Comment:
Can you assign the result of this expression to a variable on the preceding line rather than doing all inline here?

Reply:
Sure. Can you suggest a variable name?

Reply:
val_str

Reply:
@WillAyd Can I also use `np.isnan` taking advantage of the numpy import 
`val_str = lambda val: "" if np.isnan(val) else val`
`cast = lambda val: f"{val_str(val):1}"`

Reply:
I would suggest just keep it as is for this exercise